frontend/src/components/Sidebar/ActiveWorkspaces/index.jsx

```diff
@@ -82,17 +102,18 @@ export default function ActiveWorkspaces() {
                   </p>
                 </div>
                 <button
-                  onMouseEnter={() => setSettingHover(true)}
-                  onMouseLeave={() => setSettingHover(false)}
-                  onClick={() => {
+                  onClick={(e) => {
+                    e.preventDefault();
                     setSelectedWs(workspace);
                     showModal();
                   }}
+                  onMouseEnter={() => handleGearMouseEnter(workspace.id)}
+                  onMouseLeave={() => handleGearMouseLeave(workspace.id)}
                   className="rounded-md flex items-center justify-center text-white ml-auto"
                 >
                   <GearSix
-                    weight={settingHover ? "fill" : "regular"}
-                    hidden={!isActive || user?.role === "default"}
```

Comment:
We should still be hiding the button when the user role is `default` why was this removed?